eZ/Publish/Core/SignalSlot/Tests/ObjectStateServiceTest.php

```diff
@@ -55,145 +55,145 @@ public function serviceProvider()
-                array($objectState, $priority),
+                [$objectState, $priority],
                 null,
                 1,
                 'eZ\Publish\Core\SignalSlot\Signal\ObjectStateService\SetPriorityOfObjectStateSignal',
-                array(
+                [
                     'objectStateId' => $objectStateId,
                     'priority' => $priority,
-                ),
-            ),
-            array(
+                ],
+            ],
+            [
                 'deleteObjectState',
-                array($objectState),
+                [$objectState],
                 null,
                 1,
                 'eZ\Publish\Core\SignalSlot\Signal\ObjectStateService\DeleteObjectStateSignal',
-                array(
+                [
                     'objectStateId' => $objectStateId,
-                ),
-            ),
-            array(
+                ],
+            ],
+            [
                 'setContentState',
-                array($contentInfo, $objectStateGroup, $objectState),
+                [$contentInfo, $objectStateGroup, $objectState],
                 null,
                 1,
                 'eZ\Publish\Core\SignalSlot\Signal\ObjectStateService\SetContentStateSignal',
-                array(
+                [
                     'objectStateId' => $objectStateId,
                     'contentId' => $contentId,
                     'objectStateGroupId' => $objectStateGroupId,
-                ),
-            ),
-            array(
+                ],
+            ],
+            [
                 'getContentState',
-                array($contentInfo, $objectStateGroup),
+                [$contentInfo, $objectStateGroup],
                 $objectState,
                 0,
-            ),
-            array(
+            ],
+            [
                 'getContentCount',
-                array($objectState),
+                [$objectState],
                 35,
                 0,
-            ),
-            array(
+            ],
+            [
                 'newObjectStateGroupCreateStruct',
-                array('identifier'),
+                ['identifier'],
                 $objectStateGroupCreateStruct,
                 0,
-            ),
-            array(
+            ],
+            [
                 'newObjectStateGroupUpdateStruct',
-                array(),
+                [],
                 $objectStateGroupUpdateStruct,
                 0,
-            ),
-            array(
+            ],
+            [
                 'newObjectStateUpdateStruct',
-                array(),
+                [],
                 $objectStateUpdateStruct,
                 0,
-            ),
-            array(
+            ],
+            [
                 'newObjectStateCreateStruct',
-                array('identifier'),
+                ['identifier'],
                 $objectStateCreateStruct,
                 0,
-            ),
-        );
```

Comment:
Note: sorry for this big block of short-array-syntax changes. I aligned this test with extra empty array parameters and it made sense to change the whole block rather than mix short with long array syntax.